docs/victoriametrics/Release-Guide.md

```diff
@@ -13,39 +13,45 @@ aliases:
 - /release-guide/
 ---
 
-## Pre-reqs
+## PreRequisites
```

Comment:
JFYI, caution is needed when modifying the section title, as it may affect anchor links. But fortunately regarding this specific title, I did not find any links using this anchor during a global search, so it seems safe. 🎉

Reply:
yes i usually do cross ref checks across the page and across the full repo content to see if there is reference.